src/Files.App/Views/Properties/CompatibilityPage.xaml

```diff
@@ -7,114 +7,146 @@
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:helpers="using:Files.App.Helpers"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	xmlns:settingsuc="using:Files.App.UserControls.Settings"
 	xmlns:vm="using:Files.App.ViewModels.Properties"
-	Loaded="Properties_Loaded"
 	Tag="Compatibility"
 	mc:Ignorable="d">
 
 	<vm:BasePropertiesPage.Resources>
 		<ResourceDictionary>
 			<ResourceDictionary.MergedDictionaries>
 				<ResourceDictionary Source="ms-appx:///ResourceDictionaries/PropertiesStyles.xaml" />
-				<ResourceDictionary Source="/ResourceDictionaries/RightAlignedToggleSwitchStyle.xaml" />
 			</ResourceDictionary.MergedDictionaries>
 
+			<converters:EnumToBoolConverter x:Key="EnumToBoolConverter" />
 			<converters:NullToTrueConverter x:Key="NullToFalseConverter" Inverse="True" />
 
 		</ResourceDictionary>
 	</vm:BasePropertiesPage.Resources>
 
-	<ScrollViewer x:Name="RootScrollViewer">
+	<ScrollViewer>
 		<StackPanel Padding="12" Spacing="4">
 
-			<!--  Run Troubleshooter  -->
-			<settingsuc:SettingsBlockControl
-				Title="{helpers:ResourceString Name=RunCompatibilityTroubleShooter}"
-				ButtonCommand="{x:Bind CompatibilityProperties.RunTroubleshooterCommand}"
-				IsClickable="True">
-				<FontIcon
-					FontSize="14"
-					Foreground="{ThemeResource TextFillColorPrimaryBrush}"
-					Glyph="&#xE8A7;" />
-			</settingsuc:SettingsBlockControl>
+			<!--  Launch Compatibility Troubleshooter  -->
+			<StackPanel
+				Padding="16"
+				Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+				BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+				BorderThickness="1"
+				CornerRadius="{StaticResource ControlCornerRadius}"
+				Spacing="8">
+				<TextBlock Text="If this program isn't working correctly on this version of Windows, try running the compatibility troubleshooter." />
+				<Button Command="{x:Bind CompatibilityViewModel.RunTroubleshooterCommand}" Content="{helpers:ResourceString Name=RunCompatibilityTroubleShooter}" />
+			</StackPanel>
+
+			<!--  Section (Compatibility Mode)  -->
+			<TextBlock
+				x:Name="CompatibilityModeSectionTitleTextBlock"
+				Margin="0,12,0,0"
+				Style="{StaticResource BodyStrongTextBlockStyle}"
+				Text="{helpers:ResourceString Name=CompatibilityMode}" />
 
 			<!--  Compatibility Mode  -->
-			<settingsuc:SettingsBlockControl Title="{helpers:ResourceString Name=CompatibilityMode}" HorizontalAlignment="Stretch">
+			<StackPanel
+				Padding="16"
+				Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+				BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+				BorderThickness="1"
+				CornerRadius="{StaticResource ControlCornerRadius}"
+				Spacing="8">
+
+				<TextBlock Text="If this program was made for previous version of Windows, select version of Widnows to run this version of Windows as well." />
```

Comment:
A. Hard-coding English? Shouldn't this be translatable?
B. Check spelling please :⁠-⁠)

Reply:
This comes from File Explorer on Windows, but I'm gonna translate. Thanks!

Reply:
Became `Run this program in a compatibility mode`.
Thank you for pointing it out!

![image](https://github.com/files-community/Files/assets/62196528/849ae7b1-334e-4221-9efe-de61b2ff3296)